spec/models/application_letter_spec.rb

```diff
@@ -67,4 +67,12 @@
      application.status = :rejected
      expect(application).to be_valid
   end
+
+  it "calculates the correct age of applicant when event starts" do
```

Comment:
There are some edge cases which are not covered in this test. One more reason to factor this functionality out :)